Mathlib/Topology/Category/TopCat/Limits/Konig.lean

```diff
@@ -21,64 +21,37 @@ not closed points.)
 
 We give this in a more general form, which is that cofiltered limits
 of nonempty compact Hausdorff spaces are nonempty
-(`nonempty_limit_cone_of_compact_t2_cofiltered_system`).
+(`nonempty_limitCone_of_compact_t2_cofiltered_system`).
 
-This also applies to inverse limits, where `{J : Type u} [preorder J] [is_directed J (≤)]` and
-`F : Jᵒᵖ ⥤ Top`.
+This also applies to inverse limits, where `{J : Type u} [Preorder J] [IsDirected J (≤)]` and
+`F : Jᵒᵖ ⥤ TopCat`.
 
 The theorem is specialized to nonempty finite types (which are compact Hausdorff with the
 discrete topology) in lemmas `nonempty_sections_of_finite_cofiltered_system` and
-`nonempty_sections_of_finite_inverse_system` in the file `CategoryTheory.CofilteredSystem`.
+`nonempty_sections_of_finite_inverse_system` in the file `Mathlib.CategoryTheory.CofilteredSystem`.
 
 (See <https://stacks.math.columbia.edu/tag/086J> for the Set version.)
 -/
 
 -- Porting note: every ML3 decl has an uppercase letter
 set_option linter.uppercaseLean3 false
 
-open TopologicalSpace
-
 open CategoryTheory
 
 open CategoryTheory.Limits
 
-universe v u w
+universe u v w
 
 noncomputable section
 
 namespace TopCat
 
-variable {J : Type v} [SmallCategory J]
-
 section TopologicalKonig
 
-/-!
-## Topological Kőnig's lemma
-
-A topological version of Kőnig's lemma is that the inverse limit of nonempty compact Hausdorff
-spaces is nonempty.  (Note: this can be generalized further to inverse limits of nonempty compact
-T0 spaces, where all the maps are closed maps; see [Stone1979] --- however there is an erratum
-for Theorem 4 that the element in the inverse limit can have cofinally many components that are
-not closed points.)
-
-We give this in a more general form, which is that cofiltered limits
-of nonempty compact Hausdorff spaces are nonempty
-(`nonempty_limitCone_of_compact_t2_cofiltered_system`).
-
-This also applies to inverse limits, where `{J : Type u} [Preorder J] [IsDirected J (≤)]` and
-`F : Jᵒᵖ ⥤ Top`.
-
-The theorem is specialized to nonempty finite types (which are compact Hausdorff with the
-discrete topology) in lemmas `nonempty_sections_of_finite_cofiltered_system` and
-`nonempty_sections_of_finite_inverse_system` in the file `CategoryTheory.CofilteredSystem`.
-
-(See <https://stacks.math.columbia.edu/tag/086J> for the Set version.)
--/
-
-
 variable {J : Type u} [SmallCategory J]
 
-variable (F : J ⥤ TopCat)
+-- porting note: generalized `F` to land in `v` not `u`
```

Comment:
This change was already present in master, but via an implicit universe metavariable. This just documents it.